docs/user/gcp/install_upi.md

```diff
@@ -187,6 +246,8 @@ gcloud deployment-manager deployments create ${INFRA_ID}-vpc --config 01_vpc.yam
 ```
 
 ## Configure VPC variables
+Configure the variables based on the VPC created with `01_vpc.yaml`.
+If you are using a pre-existing VPC, such as a [Shared VPC][sharedvpc], set these to the .selfLink of the targeted resources.
```

Comment:
`` `.selfLink` ``

Reply:
Resolved in 657a91d789da80eb9e2e96efc4ded29f490d64a9